.ci-mgmt.yaml

```diff
@@ -5,6 +5,9 @@ parallel: 1
 timeout: 150
 generate-nightly-test-workflow: true
 providerVersion: github.com/hashicorp/terraform-provider-aws/version.ProviderVersion
+
+buildProviderPre: "VERSION=${VERSION_GENERIC} ./scripts/minimal_schema.sh"
```

Comment:
So we basically replaced makefile extensions with a generic shell based pointcut?

Reply:
Yes. It has a slight advantage that it does not interact with `make --touch` wildly, and it's documented in the cimgmt config.